src/scss/main.scss

```diff
@@ -13,3 +13,4 @@
 
 // Components
 @import 'components/component';
+@import 'components/quoteSection';
```

Comment:
we don't use camel case notation on files, please rename it to `quotes-section.scss` or just `quote.scss`. Same for the template.

Reply:
done